### ISSUE TITLE
Increase release version to 0.9.7 

### ISSUE THREAD

**Issue by pocki80** [OLSR/olsrd#62]
Hi there. When can we expect to close the "pre" stage an change version from pre-0.9.7 to 0.9.7 in olsrd/Makefile?

**Comment by fhuberts** [comment]
when I make a release.

**Comment by vchrizz** [comment]
(when) do you plan to make a release?
or what would be the typical requirement/procedure for you to make a release?

**Comment by pocki80** [comment]
> when I make a release.

can you give us an indication when you might think about releasing a new version that will no longer be "pre-" or "-dirty"?

**Comment by fhuberts** [comment]
there is no -dirty release.
since nobody volunteered to be maintainer there we be no more release

**Comment by pocki80** [comment]
Well, that's a pitty. I was looking for a way to get a better looking version identifier. jsoninfo-version shows:

```
"gitDescriptor": "v0.9.6.2-68-ga24e20a-dirty",
"version": "olsr.org - pre-0.9.7-git_a24e20a-hash_3d83b56712d547acaa693fc988197192"
```

Suffix -dirty' in gitDescriptor is added in make/hash_source.sh using "git describe --dirty". Is there a chance to fix git-repo to not being marked to be a "dirty working tree"?

Version tells to be "pre-0.9.7" what looks like a version of beta/pre-final. I know that olsrd is some state of experimental package (which is indicated by a v0.x anyway), but "pre-" will make that even stronger. So if this is still v0.9.6.2, why then calling it pre-0.9.7?

What do we need (to do?) to establish the current commit level as a new commit (0.9.7 or 0.9.6.3 or whatever)?

**Comment by fhuberts** [comment]
build from a clean workspace.
so first do a git clean -fdx and then build

**Comment by fhuberts** [comment]
I just released 0.9.7. final release probably

**Comment by pocki80** [comment]
thank you! :-)
👍
